src/main/java/pl/put/poznan/buildingInfo/model/Building.java

```diff
@@ -70,4 +87,21 @@ public Float getLighting() {
         // return temporary value in order to compile code properly
         return (float) 0.0;
     }
+
+    public float getHeatingLimit() {
+        return heatingLimit;
+    }
+
+    /**
+     * Function that returns rooms exceeding the heating limit.
+     *
+     * @return ArrayList of rooms
+     */
+    public void setHeatingLimit(float heatingLimit) {
```

Comment:
javadoc above refers to message below, right? we should change the order (setHeatingLimit - javadoc and getExceedingRooms method) 😉